### PR TITLE
[TS] LPS-162895 Fix user is not possible to send multiple emails with specific formats

### PR DESCRIPTION
**Pull request by TaiPhamLR**
Please help review this PR and leave your comments. Thanks.
Ticket: https://issues.liferay.com/browse/LPS-162895
cc: @sontruongces

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java [hunk 1]
```diff
@@ -289,8 +291,10 @@ else if (Validator.isNull(fromName)) {
...
 		}
 
-		String to = _formatContent(
-			notificationTemplate.getTo(user.getLocale()), user.getLocale(),
+		String to = _formatTo(notificationTemplate.getTo(user.getLocale()));
+
+		to = _formatContent(
+			to, user.getLocale(),
 			NotificationTermContributorConstants.RECIPIENT, notificationType,
 			object);
```
Comment (adolfopa) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java:299]: Can you move the call to _formatContent inside _formatTo? The result would be something like:
```
private String _formatTo(String to, ...) {
    ... // current code.

    return _formatContent(StringUtil.merge(emails, ","), ...);
}
```
then, the calling code would be much simpler:
```
String to = _formatTo(notificationTemplate.getTo(user.getLocale()), user.getLocale(), ...);
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java (hunk 1 above)
Comment (sontruongces) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java]: @adolfopa, I updated the code. Thanks!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:forward

**Comment by liuxingye** [comment]
ci:forward

**Comment by liuxingye** [comment]
ci:forward

**Pull request closed without merging**